prompt2model/utils/parse_json_responses.py

```diff
@@ -101,3 +101,39 @@ def parse_prompt_to_fields(
             # In case we reach maximum number of API calls, we raise an error.
             logger.error("Maximum number of API calls reached.")
             raise RuntimeError("Maximum number of API calls reached.") from last_error
+
+
+def make_request_from_prompt(prompt: str, max_api_calls: int = 5) -> str:
```

Comment:
I don't think this function name is very descriptive. Can you be more specific about what we're doing here?